.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+### Please note: the project is currently temporarily suspended from receiving direct code contributions.
+
+Please do not submit any new pull request. (It will be closed directly)
```

Comment:
```suggestion
Please do not submit any new pull requests, they will be close immediately.
```

This might read a bit better, if you agree with the intent.